tests/flows/send_specific_receipts.go

```diff
@@ -0,0 +1,239 @@
+package flows
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"math/big"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
+	teleportermessenger "github.com/ava-labs/teleporter/abi-bindings/go/Teleporter/TeleporterMessenger"
+	"github.com/ava-labs/teleporter/tests/interfaces"
+	"github.com/ava-labs/teleporter/tests/utils"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/log"
+	. "github.com/onsi/gomega"
+)
+
+var (
+	relayerFeePerMessage = big.NewInt(5)
+)
+
+func SendSpecificReceipts(network interfaces.Network) {
+	subnetAInfo, subnetBInfo, _ := utils.GetThreeSubnets(network)
+	teleporterContractAddress := network.GetTeleporterContractAddress()
+	_, fundedKey := network.GetFundedAccountInfo()
+	ctx := context.Background()
+
+	// Clear the receipt queue from Subnet B -> Subnet A to have a clean slate for the test flow.
```

Comment:
We might want to disable this test in testnet/main/run_testnet_flows.go since it can result in an arbitrary number of Teleporter messages to clear to queue.

Instead of clearing the receipt queue, could we instead:
- Call `sendSpecificReceipts` and confirm we receive them on Subnet A, as we're doing
- Check the receipt queue on Subnet B directly, and confirm that the receipts are still there

I think any way we write this test, there's going to be a race condition on Fuji since the specified receipts may be delivered by by anybody in between the call to `sendSpecificReceipts` and checking the receipt queue (however we do it) within the test. I'm sure similar Fuji-specific races exist in may of the other tests as well. Something we should keep in mind when running these tests on Fuji.

Reply:
Thanks for calling this out. I realized it when getting this flow to work correctly against external networks, but was just focused on getting it working first.

Checking that the receipts are still in the queue also isn't necessarily possible for external testnets because unrelated messages may have already removed them.

Instead, for now I opted to add a `IsExternalNetwork` function to the `Network` interface, and skip the parts of this flow that shouldn't be run against networks that aren't fully controlled by the test application itself.